charts/common/templates/addons/code-server/_container.tpl

```diff
@@ -28,6 +28,9 @@ args:
 - "{{ .Values.addons.codeserver.service.port.port }}"
 - "--auth"
 - "none"
+{{- with .Values.addons.codeserver.args }}
+{{ . }}
+{{- end }}
```

Comment:
@bjw-s do you suppose there is a better/cleaner way to achieve this here?

Reply:
I was thinking maybe you could replace the `args` section with this:

```yaml
args:
{{- range .Values.addons.codeserver.args }}
- {{ . | quote }}
{{- end }}
- "--port"
- "{{ .Values.addons.codeserver.service.port.port }}"
- {{ .Values.addons.codeserver.workingDir | default (first .Values.addons.codeserver.volumeMounts).mountPath }}
```

That way we only force the port and the startup folder, and leave everything else configurable

Reply:
much better approach, thank you!